### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by Alfred-KInyua**
### I was able to achieve the following 

Created an app.rb file that served as my console app entry-point with the following methods:

- List all books.
- List all people.
- Create a person (teacher or student, not a plain Person).
- Create a book.
- Create a rental.
- List all rentals for a given person id.

In my main.rb  I defined the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:

- Present the user with a list of options to perform.
- Lets users choose an option.
- If needed, ask for parameters for the option.
- Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    answer = person_choice
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+
+    case answer
+    when 1
+      parent_permission = parent_permission?
+      @people << Student.new(age, name, parent_permission)
```
Comment (VICTOR-CHUKWU) [inline app.rb:59]: - [ ] nice work @Alfred-KInyua , I can not create a student, it returns an error about numbers of arguments passed in your student.rb initialize method, kindly fix this
image

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    when 2
+      print 'Specialization: '
+      specialization = gets.chomp
+      @people << Teacher.new(specialization, age, name)
```
Comment (VICTOR-CHUKWU) [inline app.rb:63]: - [ ] also as in student,  kindly pass your arguments correctly as it is in the teacher.rb

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Alfred-KInyua  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (Alfred-KInyua) [reply app.rb:59]: @VICTOR-CHUKWU okay, let me fix that

Comment (Alfred-KInyua) [reply app.rb:59]: @VICTOR-CHUKWU   this parameter was the culprit  "parent_permission: true"

**Review by alexander16108 — APPROVED** [review]
Hi @Alfred-KInyua , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**